ProgrammingInC.md

```diff
@@ -0,0 +1,43 @@
+**[Back](/README.md/)**
+
+# Programming in C
+
+## Videos
+
+- [Basic C Programming](https://www.youtube.com/watch?v=rLf3jnHxSmU&list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
+
+- [C Programming & Data Structures](https://www.youtube.com/watch?v=4OGMB4Fhh50&list=PLBlnK6fEyqRhX6r2uhhlubuF5QextdCSM)
+
+- [C Programming Edureka](https://www.youtube.com/watch?v=8PopR3x-VMY)
+
+## Git Books
```

Comment:
organize it better 
list all the books under a single heading

Reply:
brother that's how I want to represent. there is no restriction on my way...its said by the owner himself(@aneagoie )